.ci/setup.sh

```diff
@@ -121,18 +124,17 @@ else  # Linux
         fi
     fi
     if [[ $SETUP_CONDA != "false" ]]; then
-        if [[ $ARCH == "x86_64" ]]; then
-            curl -sL -o conda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-        else
-            curl -sL -o conda.sh https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${ARCH}.sh
-        fi
+        curl \
+            -sL \
+            -o mambaforge.sh \
+            https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-${ARCH}.sh
     fi
 fi
 
-if [[ "${TASK}" != "r-package" ]]; then
+if [[ "${TASK}" != "r-package" ]] && [[ "${TASK}" != "r-rchk" ]]; then
```

Comment:
really interesting! I guess we've been accidentally installing conda for the `r-rchk` job, but this never failed before because the GitHub Actions environments have an installation of `conda` already?

See https://github.com/microsoft/LightGBM/runs/4909427380?check_suite_focus=

> ERROR: File or directory already exists: '/usr/share/miniconda'
If you want to update an existing installation, use the -u option.

PackageNotInstalledError: Package is not installed in prefix.
  prefix: /usr/share/miniconda
  package name: mamba

See https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-Readme.md#environment-variables....by default, GitHub Actions sets `CONDA=/usr/share/miniconda` in Linux environments.

Reply:
looks like the fix I added in https://github.com/microsoft/LightGBM/pull/4953/commits/44de0babe86aaeb0f94995725f575fbe34a22ed6 worked: https://github.com/microsoft/LightGBM/runs/4909444516?check_suite_focus=true